src/main/java/command/add/AddCommand.java

```diff
@@ -2,14 +2,70 @@
 
 import command.CommandExecutor;
 import database.EmployeeDB;
+import model.CAREERLEVEL;
+import model.CERTI;
 import model.Employee;
 import model.UserRequest;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class AddCommand implements CommandExecutor {
     @Override
-    public List<Employee> run(UserRequest request, EmployeeDB db) {
-        return null;
+    public List<Employee> run(UserRequest request, EmployeeDB db) throws Exception {
+    	if(request == null || db == null)
+    		return new ArrayList();
+    	
+    	
+    	//형식 : ADD,,,,18050301,KYUMOK KIM,CL2,010-9777-6055,19980906,PRO
+    	List<String> EmployeeInfo = request.getArguments();
+    	CAREERLEVEL careerLevel = null;
+    	if(EmployeeInfo.get(2).equals("CL1"))
```

Comment:
EmployeeInfo.get(2)가 뭔지 좀 헛갈렸는데 argument list였군요~ EmployeeInfo라고 하면 다같이 DB에서 쓰는 클래스도 Employee여서 naming이 좀 헛갈리는 면이 있는 것 같습니다. 지금 대상이 되는 데이터는 Command에 따른 User의 arguments들 이어서.. 아래처럼 변경하는건 어떨지요? 
```suggestion
    	if(arguments.get(2).equals("CL1"))
```

Reply:
좋은제안 감사드립니다. 네이밍 변경하도록 하겠습니다.... ^^